.github/workflows/cibuild.yml

```diff
@@ -61,12 +61,29 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: 3.8
+        python-version: 3.9
     - name: Install tox
       run: python -m pip install --upgrade tox
-    - name: Python 3.8 with latest astropy
-      run: tox -e py38-latest
+    - name: Python 3.9 with latest astropy
+      run: tox -e py39-test
 
+  docs:
```

Comment:
I would suggest to do docs build testing with readthedocs rather than with github actions, as the docs is served there anyway. It can/should be a separate PR though.

Reply:
Removed docs from here. It is already being built on readthedocs.